src/javascripts/components/mobile-navigation.js

```diff
@@ -63,6 +63,17 @@ MobileNav.prototype.bindUIEvents = function () {
   })
 }
 
+MobileNav.prototype.addHeadings = function () {
+  var $headings = this.$nav.querySelectorAll('.js-app-mobile-nav-subnav__link-heading')
+
+  nodeListForEach($headings, function ($headingText) {
+    const heading = document.createElement('h3')
+    heading.classList = 'app-mobile-nav-subnav__link-heading'
```

Comment:
[`Element.classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) is a read only property, which strangely only IE 11 seems to care about…

> "Assignment to read-only properties is not allowed in strict mode"

I think we'd need to use `classList.add` here?

```suggestion
    heading.classList.add('app-mobile-nav-subnav__link-heading')
```